content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx

```diff
@@ -179,6 +179,42 @@ When it is obvious what the link destination is (for example if you've explained
 [Learn more](/concepts/components/prisma-client/relation-queries)
 ```
 
+## Errors
+
+When you refer to errors, use the following guidelines.
+
+### Handling errors
+
+When you want to say that Prisma (or another system) handles an error, use the verb "handle". Avoid other terms, such as "catch".
+
+```md
+<!-- Good -->
+
+Handle exceptions and errors
+```
+
+### Error messages
+
+#### Errors returned by a CLI or API
+
+When you want to refer to errors returned by Prisma (or another system), use the verb "return".
```

Comment:
"return" is different than "throw" (which is the technical code level term)
Does this apply to the right level?

Reply:
I would not describe a function that throws an error that it returns an error. Those are two very different things in programming - `return`  vs `throw`. Return implies normal code execution, and throw implies a problem or error

Reply:
"Display" doesn't seem right to me. What about if the error is returned (sorry!) by an API endpoint? It isn't "displayed" anywhere - it's just information that comes back from the endpoint.

Reply:
This was never resolved, so I'm removing the controversial additions